source/_components/camera.amcrest.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Amcrest IP Camera"
+description: "Instructions how to integrate Amcrest IP cameras within Home Assistant."
+date: 2016-11-24 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: amcrest.png
+ha_category: Camera
+ha_release: 0.34
+---
+
+
+The `amcrest` platform allows you to integrate your [Amcrest](https://amcrest.com/) IP camera in Home Assistant.
+
+To enable your camera in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+camera:
+  platform: amcrest
+  name: Living Room
+  host: 192.168.0.15
+  port: 80
+  username: USERNAME
+  password: PASSWORD
+```
+
+Configuration variables:
+
+- **host** (*Required*): The IP address of your camera.
```

Comment:
It can be IP address or hostname. if hostname we assume that DNS or local resolution will work.  That is a good point. Let me update the doc.  Thanks @Landrash